server/test/unit/server/controllers/test_importer.py

```diff
@@ -254,6 +254,19 @@ def test_valid_as_expected(self, m_plug_call_config, m_repo_model, m_plug_api):
         imp_inst.validate_config.assert_called_once_with(
             m_repo.to_transfer_repo.return_value, m_plug_call_config.return_value)
 
+    @mock.patch('pulp.server.controllers.importer.PluginCallConfiguration')
+    def test_invalid_importer_type(self, m_plug_call_config, m_repo_model, m_plug_api):
+        m_repo = m_repo_model.objects.get_repo_or_missing_resource.return_value
+        m_plug_api.is_valid_importer.return_value = False
+        try:
+            importer.validate_importer_config(m_repo, 'm_type', 'm_conf')
+        except exceptions.PulpCodedValidationException, e:
+            pass
```

Comment:
I'd suggest to use assertRaises fin this case, like [here](https://github.com/pulp/pulp/blob/38adb192add8cd0b66940a09a8183921cace61da/server/test/unit/server/webservices/views/test_util.py#L215)
What do you think?